src/System.ComponentModel.TypeConverter/tests/TypeDescriptorTests.cs

```diff
@@ -276,6 +276,9 @@ class FooBarDerived : FooBarBase
             new Tuple<Type, Type> (typeof(ClassIBase), typeof(IBaseConverter)),
             new Tuple<Type, Type> (typeof(ClassIDerived), typeof(IBaseConverter)),
             new Tuple<Type, Type> (typeof(Uri), typeof(UriTypeConverter)),
+#if NETCORE
+            new Tuple<Type, Type> (typeof(Version), typeof(VersionConverter)),
+#endif
```

Comment:
Why did you undo my change and switch to using an ifdef?  We try hard to avoid such ifdefs, and instead prefer approaches like I suggested.

Reply:
Sorry, I did not see you did a change, I rebased on master and force pushed. Adding `Condition="'$(TargetGroup)' == 'netcoreapp'"` was not enough because of this occurence of `VersionConverter` in `TypeDescriptorTests.cs`.

What was your approach to avoid the conditional compilation?

Reply:
Not to avoid conditional compilation... to avoid the ifdef.  And also to use theories rather than iterating through an enumerable, as theories are generally the approach we prefer in this case.
https://github.com/stephentoub/corefx/commit/7d695a634336fc925a3bc5573402faae38a4ee54

Reply:
OK thanks. I reverted my `#if NETCORE` fix, cherry-picked yours, rebased on master and force-pushed.